jcg_evaluation/src/main/scala/JavaFingerprintExtractor.scala

```diff
@@ -89,32 +90,15 @@ object JavaFingerprintExtractor extends FingerprintExtractor {
                 if (config.parallel) {
```

Comment:
We would probably also want to have this parallel option for the other fingerprint extractors as well.